packages/core-react/index.d.ts

```diff
@@ -8,4 +8,16 @@ export type TextFieldProps = {
 
 declare const TextField: React.FC<TextFieldProps>;
 
-export { TextField };
+export type ButtonProps = {
+  Tag?: 'button' | 'a';
+  children?: React.ReactNode;
+  type?: 'primary' | 'secondary' | 'tertiary';
```

Comment:
I'm ok with this for now, but I think we should definitely update these to enums in the future.